### PR TITLE
Update README.md for void closure specs

### PR DESCRIPTION
**Pull request by brightredchilli**

### CODE REVIEW

**Comment by paulrolfe** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -564,6 +564,42 @@ let layout = { (view1: UIView, view2: UIView) in
...
+It is preferable to not use `Void` in closure arguments or return types, omitting parentheses where possible.
+
+#####Like this:
+```Swift
+let noArgNoReturnClosure = { doSomething() } // no arguments or return types, omit both
+let noArgClosure = { () -> Int in return getValue() } // void argument, use '()'
+let noReturnClosure = { (arg) in doSomething(with: arg) } // void return type, omit return type
+```
+
+#####Not this:
```
Comment (alexpersian) [inline README.md]: Please add an additional `#` to the `Not this:` that you've added. (6 total `#` to match the other's which are grey visually)

**Review by alexpersian — CHANGES_REQUESTED** [review]
Content looks good. Only a small visual style change requested.

File: README.md (hunk 1 above)
Comment (brightredchilli) [reply README.md]: nice catch, thanks

**Review by alexpersian — APPROVED** [review]

**Comment by alexpersian** [comment]
🚢

**Pull request merged**